src/Tribe/Aggregator/Record/Batch_Queue.php

```diff
@@ -0,0 +1,422 @@
+<?php
+/**
+ * Create a new Queue to process Batch imports.
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Events\Aggregator\Record;
+
+use DateTime;
+use DateTimeZone;
+use Exception;
+use Tribe__Date_Utils;
+use Tribe__Events__Aggregator__Record__Abstract;
+use Tribe__Events__Aggregator__Record__Activity;
+use Tribe__Events__Aggregator__Record__Queue;
+use Tribe__Events__Aggregator__Record__Queue_Cleaner;
+use Tribe__Events__Aggregator__Record__Queue_Interface;
+use Tribe__Events__Aggregator__Records;
+use Tribe__Events__Aggregator__Service;
+use Tribe__Events__Main;
+use WP_Error;
+use WP_Post;
+
+// Prevent to access the file directly.
+defined( 'WPINC' ) || die;
```

Comment:
No need, that is an old requirement of WP.

Reply:
Fantastic doing the clean up :)